staging/src/k8s.io/dynamic-resource-allocation/kubeletplugin/draplugin.go

```diff
@@ -55,7 +55,7 @@ type DRAPlugin interface {
 	// to the API server.
 	//
 	// The caller must not modify the content after the call.
-	PublishResources(ctx context.Context, resources Resources)
+	PublishResources(ctx context.Context, resources Resources) error
```

Comment:
I am not sure whether this interface will be left for custom implementations, and whether this change is appropriate